skimage/filters/_frangi.py

```diff
@@ -0,0 +1,185 @@
+import numpy as np
+
+
+__all__ = ['frangi_filter', 'hessian_filter']
+
+
+def frangi(image, scale=(1, 10), scale_ratio=2, beta1=0.5, beta2=15,
+                  black_ridges=True):
+    """Returns image filtered with frangi filter.
+
+    Calculates the eigenvectors of the Hessian to compute the likeliness of
+    an image region to vessels, according to the method described in _[1].
+
+    Parameters
+    ----------
+    image : (N, M) ndarray
+        Array with input image data.
+    scale : tuple of floats, optional
```

Comment:
@Hiyorimi are you opposed to renaming `scale` to `scale_range`? When I read `scale` I want to give a single value.